room/signaling/room.go

```diff
@@ -28,7 +28,7 @@ type HubInfo struct {
 // Hub hub
 type Hub struct {
```

Comment:
Is it okay to have `Hub` type in both `room` and `room/signaling`?

Reply:
Fixed. It's called Room in `signaling` now.